lib/index.js

```diff
@@ -13,13 +13,21 @@ var when = require('do-when');
 
 var Parsely = module.exports = integration('Parsely')
   .global('PARSELY')
-  .global('parsely')
```

Comment:
not only superflous, but never used by their [script](http://d1z2jf7jlzjs58.cloudfront.net/p.js), so not sure why this was in there.